
def foo():
print("This will give an IndentationError.")    
    
# IndentationError: expected an indented block after function definition on line 1

